.github/workflows/build-test.yml

```diff
@@ -3,14 +3,14 @@ name: build-test
 on:
   pull_request:
     paths-ignore:
-      - '**.md'    
-  push:    
+      - "**.md"
+  push:
     branches:
       - main
       - releases/*
     paths-ignore:
-      - '**.md'
-      
+      - "**.md"
```

Comment:
Hello @oscard0m . Thank you for your pull request. Could you please revert this changes ?